### ISSUE TITLE
User is not redirected to Cluster explorer on a rancher upgrade

### ISSUE THREAD

**Issue by sowmyav27** [rancher/rancher#33743]
**Rancher Server Setup**
- Rancher version: master-head commit id: `9e0d1b2`
- Installation option (Docker install/Helm Chart): Helm install
   - If Helm Chart, Kubernetes Cluster and version (RKE1, RKE2, k3s, EKS, etc): RKE1

**Information about the Cluster**
- Kubernetes version: 1.20.7

**Describe the bug**
User is not redirected to Cluster explorer on a rancher upgrade

**To Reproduce**

- On 2.5.9 , deploy a downstream cluster (not required) in the Ember UI
- Upgrade rancher to master-head.
- User is still in Ember UI on the upgraded setup

**Expected Result**
User should be redirected to Cluster Explorer, since that is the default UI from 2.6.0

**Comment by Jono-SUSE-Rancher** [comment]
I'm paraphrasing what @vincent99 said in our chat as a comment here for whoever picks this up:
The most simple effective solution would bet to drop all existing UI session (tokens) once upon the first startup of 2.6, and anybody who was logged in will go back to the login page and then into Vue from there.

**Comment by vincent99** [comment]
Yes the simplest solution I can think of is to delete all UI session tokens (there's an annotation identifying them as UI), just once, upon first startup of 2.6.

The UI can't really do much on it's own here because the user already has some existing old version loaded and running in their browser, and we haven't gone back into the past to define a way for it to know it should do anything.

**Comment by snasovich** [comment]
PR: https://github.com/rancher/rancher/pull/33849

**Comment by deniseschannon** [comment]
Available to test in v2.6.0-rc1

**Comment by snasovich** [comment]
Sending back to "Working" as we need UI change as UI stores the last path the user was at in browser storage. After they log back in they still get redirected to Ember-based UI because that was their last know path. @vincent99, please let us know when you have an update on this.
